benchbuild/__init__.py

```diff
@@ -1,24 +1,33 @@
+# pylint: disable=useless-import-alias
 """
 Public API of benchbuild.
 """
 import sys
 
-from plumbum import local
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)